internal/api/context.go

```diff
@@ -16,7 +16,7 @@ type Supervisor interface {
 
 // Store describes the interface required to persist changes made via API requests.
 type Store interface {
-	CreateRing(ring *model.Ring, installationGroups []*model.InstallationGroup) error
+	CreateRing(ring *model.Ring, installationGroup *model.InstallationGroup) error
```

Comment:
Just out of interest why did we changed this back to one group instead of multiple?